doc/man5/flux-config-exec.rst

```diff
@@ -128,15 +140,24 @@ JOB TERMINATION
 When a job is canceled or gets a fatal exception it is terminated using
 the following sequence
 
- - The job shells are notified to send ``term-signal`` to job tasks, unless
-   the job is being terminated due to a time limit, in which case ``SIGALRM``
-   is sent instead.
- - After ``kill-timeout``, any remaining shells are sent ``kill-signal``
+ - The job shells are sent ``term-signal``, unless the job is being
+   terminated due to a time limit, in which case ``SIGALRM`` is sent instead.
+   The job shells forward this signal to tasks.
+ - This is repeated with ``term-signal`` at an interval of
+   ``kill-timeout`` until ``kill-shell-delay`` has elapsed.
```

Comment:
was there consideration renaming `kill-timeout` to `term-timeout`?  It just seems a tad confusing.  Just beginning to skim the code for review, I got confused quickly.

If it's a backwards compatibility thing, could rename this and variables, but have a backwards compatible check for it?

Reply:
I considered that, but it is a `kill(2)` timeout, and also the job execution system sends `SIGTERM` only `max-term-count` times, then switches to `SIGKILL`, so `term-timeout` would be a misnomer as well. I agree this is pretty confusing.

Reply:
If this PR is a bit too much. We could consider not including the `kill-shell-delay` and just fix the documentation to report that it is always `5*kill_timeout`. We do need to keep `max-term-count` because it was requested, but adding `kill-shell-delay` may not be required (not sure if anyone would adjust it)